skimage/restoration/_denoise.py

```diff
@@ -23,7 +23,7 @@ def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
     Parameters
     ----------
     image : ndarray
-        Input image.
+        Input image (image minimum must be >0).
```

Comment:
It's not completely true: values must be >= 0, it's the maximum that has to be >0.